server/app/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (db) {
 
     // Routes that will be accessed via AJAX should be prepended with
     // /api so they are isolated from our GET /* wildcard.
-    app.use('/api', require('./routes'));
+    app.use('/', require('./routes/'));
```

Comment:
I can't see the entire file structure, but I wanted to make sure to note that app.use takes './routes/' out of the path moving forward -- this just means that the '/' you might usually include later won't be included. I'm sure you took this into account, but as I can't really see the file structure (this isn't a branch? I'm not sure why not??) I thought I would mention it.